designs/template.md

```diff
@@ -2,8 +2,7 @@
 |---------------|---------------|-------------|---|
 | @name | date | Implementable | - |
 
-Title of the Design/Proposal
-===================
+# Title of the Design/Proposal
```

Comment:
This one is proposing a change in template ideally should be in another PR
however, in this case as it is too small I think it is fine we move forward here.